unbxdSearch.js

```diff
@@ -373,33 +374,94 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
         return ("query" in params && params.query.trim().length > 0);
     };
 
-    Handlebars.registerHelper('prepareFacetName', function(txt) {
+    var slider_min = 0;
+    var slider_max = 0;
+
+    Handlebars.registerHelper('prepareFacetName', function (txt) {
         txt = txt.replace("_fq", "");
         return txt.replace("_", " ");
     });
 
-    Handlebars.registerHelper('prepareFacetValue', function(txt) {
+    Handlebars.registerHelper('prepareFacetValue', function (txt) {
         return txt.trim().length > 0 ? txt : "&nbsp;&nbsp;&nbsp;";
     });
 
+    Handlebars.registerHelper('isFacetMultilevel', function (isMultilevel, options) {
```

Comment:
Remove all unused `isFacetMultilevel` handledbars.

Reply:
It is used in the 'facettemp' handlebar template, now added in the main sdk as default too.